app/views/publications/monthly_statistics/show.html.erb

```diff
@@ -273,6 +273,70 @@
 
-    <p class="govuk-body">There are 35 candidates in the tables that were counted in last year’s publication by UCAS, as previous UCAS publications have included deferred candidates (who would not be starting their courses in the 2022 to 2023 academic year) in the totals. Deferred candidates are included in their own column here. They will also be counted in future tables for the academic year that they intend to start their course, once the provider reconfirms their application.</p>
+    <% if @presenter.deferred_applications_count.zero? %>
+      <p class="govuk-body">
+        Deferred candidates are counted in a separate column in the tables.
+      </p>
+
+      <p class="govuk-body">
+        They’re counted in the statistics report for the academic year in which they start their training after their offer is re-confirmed, as well as the year in which they apply.
+      </p>
+
+      <p class="govuk-body">
+        This means that anyone who applied this year but deferring until next year will be counted in next year’s report, as well as this one.
+      </p>
+
+      <p class="govuk-body">
+        Deferred candidates are only counted in the academic year that they start their course in once training providers re-confirm their deferred offer. Providers need to do this before candidates intend to start their training.
+      </p>
+    <% elsif @presenter.deferred_applications_count == 1 %>
+      <p class="govuk-body">
+        Deferred candidates are counted in a separate column in the tables.
+      </p>
+
+      <p class="govuk-body">
+        There is one deferred candidate in this data report that was also counted in last year’s publication by UCAS.
+      </p>
+
+      <p class="govuk-body">
+        This is because UCAS counted deferred candidates in the statistics report for the academic year in which they applied, rather than the year in which they start training.
+      </p>
+
+      <p class="govuk-body">
+        This report, on the other hand, counts deferred candidates in the year in which they start their training, as well as the year in which they apply.
+      </p>
+
+      <p class="govuk-body">
+        Anyone who applied this year but deferring until next year will be counted in next year’s report, as well as this one.
+      </p>
+
+      <p class="govuk-body">
+        Deferred candidates are only counted once training providers re-confirm their deferred offer. Providers need to do this before candidates intend to start their training. 
+      </p>
+    <% elsif @presenter.deferred_applications_count > 1 %>
+      <p class="govuk-body">
+        Deferred candidates are counted in a separate column in the tables.
+      </p>
+
+      <p class="govuk-body">
+        There are <%= pluralize(@presenter.deferred_applications_count, 'deferred candidate') %> in this data report that were also counted in last year’s publication by UCAS.
```

Comment:
Could we dedupe all this? We should have inflection for `is/are` (we already have `was/were`)

Reply:
Thanks @duncanjbrown - have had a go at de-duping this.